python_modules/libraries/dagster-dg/dagster_dg/cli/configure_editor.py

```diff
@@ -0,0 +1,47 @@
+import json
+from pathlib import Path
+
+import click
+
+from dagster_dg.cli.global_options import dg_global_options
+from dagster_dg.component import all_components_schema_from_dg_context
+from dagster_dg.config import normalize_cli_config
+from dagster_dg.context import DgContext
+from dagster_dg.utils import DgClickCommand, DgClickGroup
+from dagster_dg.utils.editor import (
+    install_or_update_yaml_schema_extension,
+    recommend_yaml_extension,
+)
+
+_DEFAULT_SCHEMA_FOLDER_NAME = ".vscode"
+
+
+@click.group(name="configure-editor", cls=DgClickGroup)
+def configure_editor_group():
+    """Commands for configuring your editor."""
+
+
+@configure_editor_group.command(name="code-location", cls=DgClickCommand)
+@dg_global_options
+@click.argument("editor", type=click.Choice(["vscode", "cursor"]))
+@click.pass_context
+def configure_editor_code_location_command(
```

Comment:
let's do it in follow up but I think we should consider a different top-level grouping

Reply:
I agree but yeah this PR is just a dumb transform of the current state

Reply:
ya hence my suggestion for follow up